website/pages/docs/upgrading/upgrade-to-1.6.0.mdx

```diff
@@ -0,0 +1,21 @@
+---
+layout: docs
+page_title: Upgrading to Vault 1.6.0 - Guides
+sidebar_title: Upgrade to 1.6.0
+description: |-
+  This page contains the list of deprecations and important or breaking changes
+  for Vault 1.6.0. Please read it carefully.
+---
+
+# Overview
+
+This page contains the list of deprecations and important or breaking changes
+for Vault 1.6.0 compared to 1.5.3. Please read it carefully.
```

Comment:
While 1.5.3 is the last one we have an upgrade guide for, the most recent 1.5 is 1.5.5. We don't really have any rules around this, but since it's a major release, maybe 1.5.x or just 1.5.
```suggestion
for Vault 1.6.0 compared to 1.5. Please read it carefully.
```